groovy-language-server/src/main/java/com/palantir/ls/groovy/GroovycWrapper.java

```diff
@@ -338,6 +428,22 @@ private SymbolInformation getVariableSymbolInformation(String parentName, String
         return symbols;
     }
 
+    private String getWorkspaceUri(String uri) {
+        if (uri.startsWith(workspaceRoot.toString())) {
```

Comment:
Just a reminder, based on the implementation here, we should probably always do the conversion of `workspaceRoot` into an absolute path as soon as it's passed in the server